klog.go

```diff
@@ -36,6 +36,8 @@ type Verbose bool
 
 func V(level Level) Verbose { return level <= maxLevel }
 
+func (v Verbose) Enabled() bool { return bool(v) }
```

Comment:
Also this one... maybe?

Reply:
The table helps people understand the mapping between go-kit and klog levels. `Enabled()` only exists to match the klog interface but has no direct mapping in go-kit so it's not worth mentioning it in the docs IMHO.